_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,248 @@
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [HTML Specification - Content model](https://html.spec.whatwg.org/#concept-element-content-model)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This [HTML `<ul>` element][] has a [descendant][] of type [HTML `<li>` element][].
+
+```html
+<ul>
+	<li>Eggs</li>
+	<li>Milk</li>
+</ul>
+```
+
+#### Passed Example 2
+
+This [HTML `<dl>` element][] contains at least one name-value group as [descendant][].
+
+```html
+<dl>
+	<dt>Physics</dt>
+	<dd>8/10</dd>
+</dl>
+```
+
+#### Passed Example 3
+
+This [HTML `<menu>` element][] has a [descendant][] of type [HTML `<li>` element][].
+
+```html
+<menu>
+	<li></li>
+</menu>
+```
+
+#### Passed Example 4
+
+This [HTML `<ol>` element][] contains at least one [descendant][] of type [HTML `<li>` element][].
+
+```html
+<ol>
+	<span role="listitem">Eggs</span>
+	<span role="listitem">Milk</span>
+</ol>
+```
+
+#### Passed Example 5
+
+This [HTML `<ol>` element][] contains at least one [descendant][] of type [HTML `<li>` element][] and contains a [HTML `<template>` element][] which follows the [content model][].
+
+```html
+<ol>
+	<li>Eggs</li>
+	<li>Milk</li>
+	<template>
+		<li>{{item}}</li>
+	</template>
+</ol>
+```
+
+#### Passed Example 6
+
+This [HTML `<dl>` element][] contains at least one name-value group as [descendant][]. That fact that it is placed in a div still follows the [content model][].
+
+```html
+<dl>
+	<div>
+		<dt>Physics</dt>
+		<dd>8/10</dd>
+	</div>
+</dl>
+```
+
+#### Passed Example 7
+
+This [HTML `<ol>` element][] contains at least one [descendant][] of type [HTML `<li>` element][].
+
+```html
+<ol>
+	<li></li>
+</ol>
+```
+
+#### Passed Example 8
+
+This [HTML `<dl>` element][] contains at least one name-value group as [descendant][]. The dive elements both have an [explicit role][] which is the [Implicit ARIA semantics]() for an HTML element included in the [content model][].
+
+```html
+<dl>
+	<div role="term">Physics</div>
+	<div role="definition">8/10</div>
+</dl>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This [HTML `<ul>` element][] doesn't contain at least one child element.
+
+```html
+<ul></ul>
+```
+
+#### Failed Example 2
+
+This [HTML `<ol>` element][] doesn't contain at least one child element. The text itself isn't an element.
+
+```html
+<ol>
+	Coming soon!
+</ol>
+```
+
+#### Failed Example 3
+
+This [HTML `<ul>` element][] only contains elements with an [explicit role][] that doesn't follow the [content model][]. Menuitems are not valid children of an unordered list.
+
+```html
+<ul>
+	<li role="menuitem">New file</li>
+	<li role="menuitem">Save file</li>
+</ul>
```

Comment:
Technically, this passes the rule. It satisfies Expectation 1 since each child is matching the condition of the first bullet (it's a `li`, allowed by content model inside a `ul`).

Reply:
Right you are, fixing this.